src/main/webapp/i18n/de/user-management.json

```diff
@@ -31,6 +31,12 @@
                 "internal": "Intern",
                 "external": "Extern"
             },
+            "registrationNumber": {
+                "title": "Die Registrierungsnummer ({{ num }})",
+                "tooltip": "Es filtert die Benutzer anhand ihrer Registrierungsnummer",
```

Comment:
I suggest to use a similar translation as we use in the other filters: `Filter die Nutzer:innen anhand ihrer Registrierungsnummer`

Reply:
Thanks Dominik! I changed the translation accordingly.